config/default.yml

```diff
@@ -22,13 +22,23 @@ metrics:
     enabled: false
     minimum: 70
     upload_report: true
-    report_dir: 'coverage'
+    report_dir: 'slather-output'
+    uploader_key: <%= ENV['CODE_STATS_S3_KEY'] %>
+    uploader_secret: <%= ENV['CODE_STATS_S3_SECRET_KEY'] %>
+    uploader_region: <%= ENV['CODE_STATS_S3_REGION'] %>
+    uploader_bucket: <%= ENV['CODE_STATS_S3_BUCKET'] %>
+  swiftlint:
+    name: 'Swiftlint'
+    enabled: false
+    minimum: 70
+    upload_report: true
+    report_dir: 'swiftlint-output'
     uploader_key: <%= ENV['CODE_STATS_S3_KEY'] %>
     uploader_secret: <%= ENV['CODE_STATS_S3_SECRET_KEY'] %>
     uploader_region: <%= ENV['CODE_STATS_S3_REGION'] %>
     uploader_bucket: <%= ENV['CODE_STATS_S3_BUCKET'] %>
   tailor:
-    name: 'Code Quality'
+    name: 'Tailor'
```

Comment:
I just didn't want `Tailor` and `Swiftlint` to say `Code Quality`.

I could call them: 

- `Code Quality (Tailor)`
- `Code Quality (Swiftlint)`
- `Code Coverage (Slather)`

What do you think?